actor/echo/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "echo"
+version = "0.1.0"
+authors = [ "wasmcloud Team" ]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+async-trait = "0.1"
+console_log = "0.2"
+futures = "0.3"
+log = "0.4"
+serde_bytes = "0.11"
+serde_json ="1.0"
+serde = {version = "1.0", features = ["derive"]}
+
+[dependencies.wasmcloud-interface-httpserver]
+version = "0.1"
+path = "../../../interfaces/httpserver/rust"
+
+[dependencies.wasmbus-rpc]
+version = "0.3"
+path = "../../../weld/rpc-rs"
```

Comment:
If I read this repo correctly, it seems to depend on the interfaces repository. I think this may be better as a git dependency at least until we have published versions of these crates

Reply:
submitted issues for getting those repos set up with CI for publishing to crates.io

Reply:
I have published some of the common dependencies, to avoid having to use relative paths. I should have commented these out here.